.github/workflows/config.yml

```diff
@@ -0,0 +1,35 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs
+
+name: RideOn-frontend
+
+on:
+  push:
+    branches: [ "main", "*", "*/*", "**" ]
```

Comment:
It is good if pipeline can triggered on the branch also, so that we can make it green before raising a MR, also if we are doing some changes of CI/CD, we can see the changes in the pipeline on branch itself. What do you think?